### ISSUE TITLE
Element will not accept security key, even if reset

### ISSUE THREAD

**Issue by johnspackman** [element-hq/element-web#28687]
### Steps to reproduce

I have been using Element for years, signing in with Github.  I've transferred to a new Mac, but cannot get Element to accept any security keys.

Element is running - I signed out, chose to sign in with Github (no idea why, but itr was signing in with a matrix account in my name).  It asks for a security key, but will not accept it.

In the web browser (which works), I go to my icon -> Security & Privacy -> Encryption -> Secure Backup -> Reset.  Create brand new keys, but when uploaded to Element the "Continue" button does not work, and if pasted in then Element says "Wrong Security Key"

#### What did you expect?
To be able to log in

#### What happened instead?
Element refused

### Operating system

MacOS

### Application version

1.11.87

### How did you install the app?

https://element.io/download

### Homeserver

Github

### Will you send logs?

No

**Comment by richvdh** [comment]
Sounds like https://github.com/element-hq/element-web/issues/27806

**Comment by richvdh** [comment]
also https://github.com/element-hq/element-web/issues/20256

**Comment by richvdh** [comment]
I don't think we can do much with this without debug logs from within the client.
